### PR TITLE
fixed no shown grids on khmer site topic page issue #422

### PR DESCRIPTION
**Pull request by S-mardii**

### CODE REVIEW

File: archive-topic.php [hunk 1]
```diff
@@ -23,15 +23,17 @@
...
+				$taxonomy_categories_1 = array("Agriculture and fishing", "Disaster and emergency response", "Environment and natural resources", "Extractive Industries", "Land");
```
Comment (S-mardii) [inline archive-topic.php]: I listed only the top level categories.

**Comment by acorbi** [comment]
@S-mardii @Huyeng If I understand it correctly, this PR not only fixes the issue of the grid not being shown on the khmer site but also reduces the list of the topics shown to the top-tier ones, correct? I need to check with @DBishton since this affects editorial...

**Comment by S-mardii** [comment]
@acorbi I have test the code on my local and it's shown the same number of topics in the list. The thing we are trying to do is to find the very top level category of each post's category.

**Comment by acorbi** [comment]
@S-mardii I see, that is an **excelent** improvement on the code. bravo. :+1: :clap: . Im merging this now

**Pull request merged**